src/solvers/flattening/boolbv.h

```diff
@@ -151,7 +151,7 @@ class boolbvt:public arrayst
   virtual bvt convert_bv_typecast(const typecast_exprt &expr);
   virtual bvt convert_add_sub(const exprt &expr);
   virtual bvt convert_mult(const exprt &expr);
-  virtual bvt convert_div(const div_exprt &expr);
+  virtual bvt convert_div(const div_exprt &div_expr);
   virtual bvt convert_mod(const mod_exprt &expr);
```

Comment:
I would make that parameter declaration anonymous; it's obvious from the type that this is a division expression.

Reply:
It's obvious enough in the argument, but when reading the function implementation I like having a little reminder of what an expression is supposed to represent in the variable name.